packages/react-email/package.json

```diff
@@ -30,14 +30,14 @@
     "@babel/core": "7.24.5",
     "@babel/parser": "7.24.5",
     "chalk": "4.1.2",
-    "chokidar": "3.6.0",
+    "chokidar": "^4.0.1",
```

Comment:
are these updates required? if not, can we move them to a separate PR?

Reply:
This is a bit confusing, but because I merged instead of rebasing once, the files changed are a bit messy. Every time I do rebease it removes these extra changes, but for some reason they came back after? They should be gone now, just 77 instead of 150 files changed